doc/usermanual/lua/lua_api.xml

```diff
@@ -7564,7 +7564,7 @@ This function is recursion-safe and can be used to dump _G if needed.</para>
 <para>The widgets contained by the box</para>
 <para>You can append widgets by adding them at the end of the list</para>
 <para>You can remove widgets by setting them to nil</para>
-
+<para>You can replace widgets at specific index by simply assignment</para>
```

Comment:
I'd reword it like this: "You can replace widgets at a specific index by a simple assignment".